v2/internal/util/postgresql/psqluser.go

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package psqluser
```

Comment:
minor: package name should match folder name == `postgresql`.

Also this file should probably just be `psql.go` or `postgresql.go`, to match to structure in mysql where this is called `mysql.go`

Reply:
Fixed package name and file name is now postgresql.go